models/hourly_spend.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
```

Comment:
can you add a note about any nuances / assumptions / caveats? any values subject to change upon rebuild (i.e. cloud services spend, storage)?

Reply:
Nothing changes, existing tests still pass.